velocity/tool/src/java/org/sakaiproject/cheftool/VelocityPortletPaneledAction.java

```diff
@@ -205,7 +205,7 @@ public static void addAlert(SessionState state, String message)
 		String soFar = (String) state.getAttribute(STATE_MESSAGE);
 		if (soFar != null)
 		{
-			soFar = soFar + "\n\n" + message;
+			soFar = soFar + "<br/>" + message;
```

Comment:
Is this ok? This is used outside of resources.

Reply:
I'll shift it to the types.properties file, there is other HTML in there.